standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+Additionally, the Endpoint Profile cannot be inferred from the URL.  Both `https_web` and `https_spiffe` use normal HTTPS URLs with the same requirements. There is no secure method to distinguish them.  It is also insufficient to attempt `https_web` and fall back to `https_spiffe`, or vice versa, for similar reasons as outlined above: the ability to host a file with Web PKI at a particular HTTPS endpoint is not equivalent from a security perspective to the ability to host it with a valid SPIFFE SVID.
+
+### 7.3. Preserving the `<Trust Domain, Bundle>` Binding
+When authenticating an SVID, the verifier must use only the bundle for the trust domain the SPIFFE ID resides in. If we simply pooled all the bundles and accepted an SVID as long as it validates against some bundle, then trust domains could easily impersonate each other’s identities.  Another way to say this is that bundles are scoped to a particular trust domain.
+
+Since bundles are not self-describing in terms of trust domain and are also self-issued, it is critical that the binding between trust domain name and bundle endpoint, configured as part of a SPIFFE federation relationship, is then translated to a binding between trust domain name and bundle when bundles are stored and disseminated. This requirement is unlike traditional Web PKI where a single root certificate store is used to validate all certificates, regardless of which CA system actually issued the certificate being validated.
+
+### 7.4. Trustworthiness of the Bundle Endpoint Server
+The trustworthiness and integrity of a bundle endpoint server is essential in ensuring the security of the trust domain that the bundle represents. This includes not just the bundle endpoint server itself, but also the platform on which it runs, as well as any entity with administrative control over it or its platform.
+
+While this fact may appear self evident, there are situations in which it may be less obvious. For example, in the case of a non-self-serving bundle endpoint wherein trust domain A serves a bundle for trust domain B, trust domain B is implicitly trusting trust domain A and its administrators to serve the correct bundle contents. Similarly, if serving a bundle from a hosting platform such as AWS S3, operators of the trust domain that the bundle in question represents are implicitly trusting AWS to serve the correct bundle contents.
+
+When choosing a location from which a SPIFFE bundle will be served, it is important to consider the trustworthiness of the parties involved.
+
+### 7.5. Authenticity of the Bundle Endpoint
+Ensuring the authenticity of the bundle endpoint is of paramount importance. This cannot be stated strongly enough. This section explores a number of considerations that should be made in ensuring the authenticity of bundle endpoints.
+
+#### 7.5.1. Bundle Endpoint URL Redirection
```

Comment:
The text in this section is an accepted suggestion from @spikecurtis. I have personally reviewed it and think it looks good, but it would benefit from further review by @justinburke and @azdagron

Reply:
:shipit: